plugins/GSdx/GSRendererHW.cpp

```diff
@@ -250,14 +251,14 @@ void GSRendererHW::RoundSpriteOffset()
 		}
 #endif
 
-		int   oy  = m_context->XYOFFSET.OFY;
-		int   Y0  = v[i].XYZ.Y   - oy;
-		int   Y1  = v[i+1].XYZ.Y - oy;
-		int   Ly  = (v[i+1].XYZ.Y - v[i].XYZ.Y);
-		float ay0 = alpha0(Ly, Y0, Y1);
-		float ay1 = alpha1(Ly, Y0, Y1);
-		int   ty0 = Interpolate_UV(ay0, v[i].V, v[i+1].V);
-		int   ty1 = Interpolate_UV(ay1, v[i].V, v[i+1].V);
+		int	   oy  = m_context->XYOFFSET.OFY;
+		int	   Y0  = v[i].XYZ.Y   - oy;
+		int	   Y1  = v[i+1].XYZ.Y - oy;
+		int	   Ly  = (v[i+1].XYZ.Y - v[i].XYZ.Y);
+		float  ay0 = alpha0(Ly, Y0, Y1);
+		float  ay1 = alpha1(Ly, Y0, Y1);
+		uint16 ty0 = (uint16)Interpolate_UV(ay0, v[i].V, v[i+1].V);
+		uint16 ty1 = (uint16)Interpolate_UV(ay1, v[i].V, v[i+1].V);
```

Comment:
It'd be better to make ``Interpolate_UV()`` to return an ``uint16``. value, this way you avoid all those casts.